cookiecutter/tap-template/hooks/post_gen_project.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+from pathlib import Path
+
+
+BASE_PATH = Path('{{cookiecutter.library_name}}')
+
+
+def delete_folder(pth: Path):
+    for sub in pth.iterdir():
+        if sub.is_dir():
+            delete_folder(sub)
+        else:
+            sub.unlink()
+    pth.rmdir()
```

Comment:
Why not use `shutil.rmtree`?

Reply:
If I'm being honest, it's because I've been indoctrinated to never use `shutil` and got used to doing like this. 
It's much more concise with it. Thanks for pointing it out!